comp/core/tagger/collectors/workloadmeta_extract.go

```diff
@@ -708,6 +721,8 @@ func buildTaggerEntityID(entityID workloadmeta.EntityID) string {
 		return fmt.Sprintf("process://%s", entityID.ID)
 	case workloadmeta.KindKubernetesDeployment:
 		return fmt.Sprintf("deployment://%s", entityID.ID)
+	case workloadmeta.KindHost:
+		return fmt.Sprintf("host_tags://%s", entityID.ID)
```

Comment:
I would suggest to call this entity `host` or `localhost`. because we might extend this entity to store other information, like the host SBOM.